include/xtensor-io/xgzip.hpp

```diff
@@ -63,7 +63,13 @@ namespace xt
                             return uncompressed_buffer;
                     }
                     have = GZIP_CHUNK - zs.avail_out;
-                    uncompressed_buffer.insert(std::end(uncompressed_buffer), out, out + have / sizeof(T));
+                    // not possible to insert in a xt::svector currently
+                    //uncompressed_buffer.insert(std::end(uncompressed_buffer), out, out + have / sizeof(T));
+                    // so just loop for now
+                    for (int i = 0; i < have / sizeof(T); i++)
+                    {
+                        uncompressed_buffer.push_back(out[i]);
+                    }
```

Comment:
I opened https://github.com/xtensor-stack/xtensor/issues/2164 to keep track of this limitation.

Reply:
Should I use `insert` now that it is implemented in https://github.com/xtensor-stack/xtensor/pull/2165?